openpathsampling/tests/test_pyemma.py

```diff
@@ -0,0 +1,92 @@
+import openpathsampling as paths
+import numpy as np
+import os
+import pytest
+from itertools import combinations
+from .test_helpers import data_filename
+from openpathsampling.engines.openmm.tools import trajectory_from_mdtraj
+pyemma = pytest.importorskip("pyemma")
+mdtraj = pytest.importorskip("mdtraj")
+
+
+class TestPyEMMA(object):
+    def setup(self):
+        self.md_trajectory = mdtraj.load(data_filename("ala_small_traj.pdb"))
+        self.ops_trajectory = trajectory_from_mdtraj(self.md_trajectory)
+        # output of f.pairs(f.select_Backbone()) in the pyemma_generator
+        self.pairs = list(combinations([4, 6, 8, 14, 16, 18], 2))
+
+        def pyemma_generator(f):
+            f.add_inverse_distances(f.pairs(f.select_Backbone()))
+
+        self.cv = paths.collectivevariable.PyEMMAFeaturizerCV(
+            'pyemma',
+            pyemma_generator,
+            topology=self.ops_trajectory[0].topology
+            ).with_diskcache()
+        self.storage = paths.Storage('delete.nc', 'w')
```

Comment:
Use the [pytest `tmpdir` fixture](https://docs.pytest.org/en/6.2.x/tmpdir.html#the-tmpdir-fixture) here and you can save yourself the teardown hassle

Reply:
That one does not seem to catch on `setup` functions (`setup(self, tmpdir)` triggers a `missing 1 required positional argument` error). 

So then I would have to boilerplate the storage creation for every test. I think this setup/teardown is more clear